React/CoreModules/RCTDevSettings.mm

```diff
@@ -196,6 +196,27 @@ - (void)initialize
                          queue:dispatch_get_main_queue()
                      forMethod:@"reload"];
   }
+    
+#if RCT_DEV_MENU
+  if (self.bridge) {
+    RCTBridge *__weak weakBridge = self.bridge;
+    _bridgeExecutorOverrideToken = [[RCTPackagerConnection sharedPackagerConnection]
```

Comment:
This reuses the same instance variables as the "reload" action above, which means cleanup will fail. Please add other variables for `_bridgeExecutorOverrideToken` and `reloadToken

Reply:
Nice catch! Fixed 👍